Neos.ContentRepository.TestSuite/Classes/Behavior/Features/Bootstrap/ProjectedNodeTrait.php

```diff
@@ -81,23 +81,24 @@ public function iGetTheNodeAtPath(string $serializedNodePath): void
     public function iExpectANodeIdentifiedByXToExistInTheContentGraph(string $serializedNodeDiscriminator): void
     {
         $nodeDiscriminator = NodeDiscriminator::fromShorthand($serializedNodeDiscriminator);
-        $this->currentContentStreamId = $nodeDiscriminator->contentStreamId;
-        $this->initializeCurrentNodeFromContentGraph(function (ContentGraphInterface $contentGraph) use ($nodeDiscriminator) {
-            $currentNodeAggregate = $contentGraph->findNodeAggregateById(
-                $nodeDiscriminator->nodeAggregateId
-            );
-            Assert::assertTrue(
-                $currentNodeAggregate?->occupiesDimensionSpacePoint($nodeDiscriminator->originDimensionSpacePoint),
-                'Node with aggregate id "' . $nodeDiscriminator->nodeAggregateId->value
-                . '" and originating in dimension space point "' . $nodeDiscriminator->originDimensionSpacePoint->toJson()
-                . '" was not found in content stream "' . $nodeDiscriminator->contentStreamId->value . '"'
-            );
-
-            return $currentNodeAggregate->getNodeByOccupiedDimensionSpacePoint($nodeDiscriminator->originDimensionSpacePoint);
-        });
+        $contentGraphFinder = $this->currentContentRepository->projectionState(ContentGraphFinder::class);
+        $contentGraphFinder->forgetInstances();
+        $workspaceName = $this->currentContentRepository->getWorkspaceFinder()->findOneByCurrentContentStreamId(
+            $nodeDiscriminator->contentStreamId
+        )->workspaceName;
+        $contentGraph = $this->currentContentRepository->getContentGraph($workspaceName);
+        $currentNodeAggregate = $contentGraph->findNodeAggregateById(
+            $nodeDiscriminator->nodeAggregateId
+        );
```

Comment:
ah i see these lines mean that we still write tests like

> Then I expect a node identified by cs-identifier;nody-mc-nodeface;{"language":"mul"} to exist in the content graph

but dont we want to have `live;nody-mc-nodeface;{"language":"mul"}` in the future?
Maybe we can have it legacy for cs ids and change this to workspace names if possible?

Reply:
Not entirely sure yet; Here we want to test as low level as possible to be able to track changes across content streams in the same workspace. So for now I'd say this should stay this way.

Reply:
But lets merge this first and do the other in a new pr i guess? (If even at all now)

Reply:
Yes, please ^^